web/modules/custom/pineapple_general/pineapple_general.module

```diff
@@ -93,6 +93,13 @@ function pineapple_general_form_alter(&$form, \Drupal\Core\Form\FormStateInterfa
           }
         }
       }
+      if (key_exists('form_id', $form_state->getBuildInfo())) {
```

Comment:
Again, why are you checking if the form Id key exists when your entire switch statement is based on the form id and you're inside it?

Reply:
I'm sorry I didn't notice. The entire function was somewhere else and when I moved it, I accidentally left those if statements there.

Reply:
It's done

Reply:
Read what the code you’re copying does and also the code in surrounding areas these kinds of mistakes shouldn’t be happening if you’re paying attention to what you’re doing.